### PR TITLE
UX/UI Stuff

### PR DESCRIPTION
**Pull request by JakubPlatzek**

### CODE REVIEW

File: src/pages/movie.tsx
Comment (RokasGit) [inline src/pages/movie.tsx:1]: again not sure why you deleted default values?

File: src/pages/find-users.tsx
Comment (RokasGit) [inline src/pages/find-users.tsx:1]: if u deleted this, as I unterstand you included it in mvp, did you set default value to No users found if the returned list is empty?

Comment (JakubPlatzek) [reply src/pages/find-users.tsx:1]: No

File: src/pages/movie.tsx
Comment (JakubPlatzek) [reply src/pages/movie.tsx:1]: Because it wasn't working

File: src/components/movie-card.tsx
[the remark this answers is not in the document]
Comment (JakubPlatzek) [reply src/components/movie-card.tsx:1]: It was writing NaN or first letter instead of the whole thing

**Review by RokasGit — APPROVED** [review]

**Pull request merged**